cds_ils/templates/invenio_app_ils_circulation/mail/cds_active_loans.html

```diff
@@ -0,0 +1,28 @@
+{% block subject %}
+InvenioILS: Active loans of patron "{{ patron.name }}" with pid "{{ patron.pid }}".
+{% endblock %}
+
+{% block body_plain %}
+Dear librarians,
+
+when removing users who are no longer registered in the system, it has been found a patron with active loans.
```

Comment:
Maybe something like ` a patron has been found with active loans` sounds better?

Reply:
it will be approved or rewritten by the library so I suggest its better not to change content for now